src/NServiceBus.Core/Pipeline/RootContext.cs

```diff
@@ -2,11 +2,13 @@ namespace NServiceBus
 {
     using System;
     using System.Threading;
+    using Extensibility;
 
+    //TODO rename to PipelineRootContext?
     class RootContext : BehaviorContext
     {
-        public RootContext(IServiceProvider builder, MessageOperations messageOperations, IPipelineCache pipelineCache, CancellationToken cancellationToken = default)
-            : base(null, cancellationToken)
+        public RootContext(IServiceProvider builder, MessageOperations messageOperations, IPipelineCache pipelineCache, CancellationToken cancellationToken, ContextBag parentContext = null)
```

Comment:
made the cancellation token mandatory, since we always pass one when creating a RootContext